### PR TITLE
update version and url in homebrew Formula on release

### PR DESCRIPTION
**Pull request by davidbloss**
## Issues

[#216](https://github.com/OpsLevel/team-platform/issues/216)

## Changelog

Finishing tonight, just need to include updated path to released tar.gz

- [X] List your changes here
- [ ] Make a `changie` entry, N/A release only

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -79,6 +79,23 @@ jobs:
...
+        uses: actions/checkout@v4
+        with:
+          repository: OpsLevel/homebrew-tap
+          token: ${{ secrets.ORG_GITHUB_TOKEN }}
+          path: homebrew-tap
+      - name: Update homebrew
+        working-directory: homebrew-tap
+        run: |
+          cd Formula
+          awk -v ver="${RELEASE_VERSION}" '/version/, /tags/ {sub(/[0-9]+\.[0-9]+\.[0-9]+/, ver)} {print}' kubectl.rb > tmp.rb
```
Comment (davidbloss) [inline .github/workflows/release.yaml:92]: awk breakdown: modifies [kubectl.rb](https://github.com/OpsLevel/homebrew-tap/blob/main/Formula/kubectl.rb) 

awk 
`-v ver="${RELEASE_VERSION}"` // set `ver` as variable equal to ${RELEASE_VERSION}
`'/version/, /tags/` // start on line with "version", stop on  line with "tags"
`{sub(/[0-9]+\.[0-9]+\.[0-9]+/, ver)}` // substitute regex pattern for version numbers with `ver` variable
`{print}' kubectl.rb` // print file with substitutions to kubectl.rb

Comment (rocktavious) [reply .github/workflows/release.yaml:92]: Not that i don't love the ingenuity but its the complexity and possible brittleness i'm worried about.

If we are going to rip homebrew out of go releaser maybe we should also look for a formula building replacement.

It looks like brew has some stuff built in and there are a number of github actions out there to bump the version of formula in a tap.

``` brew bump-formula-pr --help```

IIRC brew comes pre-installed on the macos github actions runners and it is easily installable on linux.

Comment (davidbloss) [reply .github/workflows/release.yaml:92]: It makes sense _something_ would be out there for this 😄 
[link for later](https://github.com/Homebrew/brew/blob/master/docs/How-To-Open-a-Homebrew-Pull-Request.md)

Comment (rocktavious) [reply .github/workflows/release.yaml:92]: Reading more deeply it seems `brew bump-formula-pr` is used for updating homebrew core taps and it doesn't seem to work with tap's (at least not that I could understand)

**Pull request merged**